modules/src/core/ics04_channel/handler/recv_packet.rs

```diff
@@ -98,16 +100,16 @@ pub fn process(ctx: &dyn ChannelReader, msg: &MsgRecvPacket) -> HandlerResult<Pa
         }
 
         PacketResult::Recv(RecvPacketResult::Success(RecvPacketSuccess {
-            port_id: packet.source_port.clone(),
-            channel_id: packet.source_channel,
+            port_id: packet.destination_port.clone(),
+            channel_id: packet.destination_channel,
             seq: packet.sequence,
             seq_number: next_seq_recv.increment(),
```

Comment:
Can we rename `seq` and `seq_number`, and document what each does? It's hard for me to tell if this is correct because I'm confused as to exactly what each achieves.